source/dbo/Views/Power BI Reporting/vwHangfireJobs.sql

```diff
@@ -0,0 +1,12 @@
+﻿CREATE VIEW [dbo].[vwHangfireJobs]
+	AS
+	SELECT 
+		SUBSTRING(Q1.InvocationData, 25, EndOfString-25) AS JobName, 
+		Q1.StateName, 
+		Q1.CreatedAt 
+	FROM
+		(SELECT StateName, CreatedAt, CHARINDEX(',',InvocationData, 1) AS EndOfString, InvocationData
+		FROM [$(NTBS)].[HangFire].[Job]) AS Q1
```

Comment:
I don't think this is going to play well with the on-demand migration records in Hangfire. An example of one is 

```
{"t":"ntbs_service.DataMigration.INotificationImportService, ntbs-service","m":"BulkImportByDateAsync","p":["Hangfire.Server.PerformContext, Hangfire.Core","System.Int32","System.DateTime, mscorlib","System.DateTime, mscorlib"]}
```
And an example row this produces is:
| JobName | StateName | CreatedAt |
| ---- | --------- | ------ |
| `igration.INotificationImportService` | `Deleted` | `2021-06-15 11:06:27.920` |

which I don't think was the intended result

Reply:
@benjimarshall Yeah I'll add a WHERE clause to only look for records where the Invocation data includes 'ntbs_service.Jobs.'